releases/CHANGELOG.md

```diff
@@ -4,6 +4,23 @@ All notable changes to `grf` will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [1.1.0] - 2020-03-12
+
+### Changed (breaking)
+**IMPORTANT** These changes might cause small differences in results compared to previous releases, even if the same random seed is used.
+- Performance improvement: remove an unnecessary splitting rule loop. Note: this may cause very small differences from earlier versions because the order potential splits are evaluated in changes to the proper sort order. [#592](https://github.com/grf-labs/grf/pull/592)
```

Comment:
I am having a little trouble understanding the description, maybe we could just say "This may cause very small differences from earlier versions because it changes the order in which potential splits are evaluated."